### DISCUSSION TITLE
I am unable to install Windows VM due to no network connection

### DISCUSSION THREAD

**Discussion by siammais10** [utmapp/UTM#3928] [UTM Help (macOS)]
**Describe the issue**
Each time I try to install windows virtual machine using VHDX file, I get stuck on the network step. It says it can't continue until I connect to a network, but it's not detecting my computer's network. I do have it linked to a virtual network card. How do I get past this step? 

**Configuration**
* UTM Version: 3.1.5 (53)
* OS Version: 12.0.1
* Intel or Apple Silicon? Silicon

**Comment by wordskey** [comment]
i have the same problem

**Comment by AAFC992** [comment]
I have this issue to.

UTM Version: 3.1.5 (53)
OS Version: 12.3.1
Intel or Apple Silicon? Silicon

**Comment by AAFC992** [comment]
> 

I got it.
1-Turn off your VM
2-Go to edit options 
3-Clic on Input section
4- Change USB Support, Maybe you have USB 2, set USB 3
5-Turn on you VM and set the new device (Right conner)

**Comment by wordskey** [comment]
for device driver what can i do? miss wifi and other device thank

**Comment by Sunoo** [comment]
Follow the instructions under “Networking does not work” on https://mac.getutm.app/gallery/windows-11-arm
